framework/source/class/qx/bom/Cookie.js

```diff
@@ -62,7 +62,7 @@ qx.Bootstrap.define("qx.bom.Cookie",
         end = document.cookie.length;
       }
 
-      return decodeURI(document.cookie.substring(len, end));
+      return unescape(document.cookie.substring(len, end));
     },
```

Comment:
ah this is a revert to your own patch series ... you definitely need to squash these bevor we can merge ...